### PR TITLE
Refine log level of querynode and querycoord

### PR DESCRIPTION
**Pull request by czs007**
Signed-off-by: zhenshan.cao <zhenshan.cao@zilliz.com>

### CODE REVIEW

File: internal/querycoordv2/server.go [hunk 1]
```diff
@@ -516,7 +516,7 @@ func (s *Server) recoverCollectionTargets(ctx context.Context, collection int64)
 	err := s.targetMgr.UpdateCollectionNextTarget(collection)
 	if err != nil {
 		msg := "failed to update next target for collection"
-		log.Error(msg, zap.Error(err))
+		log.Warn(msg, zap.Error(err))
```
Comment (yah01) [inline internal/querycoordv2/server.go:519]: This has to be a error, recovery failure will lead to panic later

**Review by yah01 — CHANGES_REQUESTED** [review]

File: internal/querynode/flow_graph_filter_dm_node.go [hunk 2]
```diff
@@ -118,7 +118,7 @@ func (fdmNode *filterDmNode) Operate(in []flowgraph.Msg) []flowgraph.Msg {
 				metrics.QueryNodeConsumeCounter.WithLabelValues(strconv.FormatInt(paramtable.GetNodeID(), 10), metrics.DeleteLabel).Add(float64(proto.Size(&resMsg.DeleteRequest)))
 			}
 		default:
-			log.Warn("invalid message type in filterDmNode",
+			log.Info("invalid message type in filterDmNode",
```
Comment (jiaoew1991) [inline internal/querynode/flow_graph_filter_dm_node.go:121]: why is different with `in == nil`

Comment (czs007) [reply internal/querynode/flow_graph_filter_dm_node.go:121]: It is certain that there are some messages (such as ddl messages) in the dmChannel that the query does not process.
So it seems weird to keep warning?
Under normal circumstances, nil will not be passed in

File: internal/querycoordv2/server.go (hunk 1 above)
Comment (czs007) [reply internal/querycoordv2/server.go:519]: ok, get

**Pull request closed without merging**